--- a/drivers/gpu/drm/panel/panel-visionox-vtdr6130-cog.c	2026-07-27 22:28:33.556624923 -0700
+++ b/drivers/gpu/drm/panel/panel-visionox-vtdr6130-cog.c	2026-07-27 22:28:08.067037453 -0700
@@ -101,10 +101,10 @@
     mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x51, 0x07, 0xFF);
 	mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x6F, 0x02);
 	mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x36, 0x02);
-	mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x59, 0x09);
+	mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x59, 0x00);
 
 	if (cur_vrefresh == 120) {
-		mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x6C, 0x00);
+		mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x6C, 0x01);
 	}
 	else {
 		mipi_dsi_generic_write_seq_multi(&dsi_ctx, 0x6C, 0x01);
